UI/NpcList/NPCRow.gd

```diff
@@ -64,8 +64,13 @@ func _on_Forget_pressed():
 
 
 func _on_ShowNPC_pressed():
+	GM.ui.clearCharactersPanel()
 	GM.main.playAnimation(StageScene.Duo, "stand", {npc=_npcID})
-
+	
+	var character = GlobalRegistry.getCharacter(_npcID)
+	GM.ui.getCharactersPanel().addCharacter(_npcID)
+	character.addEffect(StatusEffect.SexEngineLikes)
+	GM.ui.updateCharactersInPanel()
```

Comment:
I wonder if something will break if you look at a character and then back out of the menu while they are still in the characters panel

Reply:
Your buttons clear everything :) If you click "back" it's all good, but I don't know if there are other ways of getting out of this menu.